### PR TITLE
ci: pretty unit test

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,7 +66,7 @@ endif
 .PHONY: test
 test: codegen
 	@oomplay init $(TEST_PLAYGROUNDS) -j 16
-	@go test -race -coverprofile=coverage.out -covermode=atomic ./...
+	@gotestsum -f testname  -- -coverprofile=cover.out -race -covermode=atomic ./...
```
Comment (wfxr) [inline Makefile:69]: Install it from binary ?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -71,6 +71,9 @@ jobs:
         pkg=oomplay-musl_${v}_amd64.deb
         wget https://github.com/oom-ai/oomplay/releases/download/v${v}/${pkg} &&
         sudo dpkg -i ${pkg}
+    - name: Install gotestsum
```
Comment (wfxr) [inline .github/workflows/ci.yml:74]: Install it from binary ?

Comment (lianxmfor) [reply .github/workflows/ci.yml:74]: Is it to standardize the installation in the ci?

Comment (wfxr) [reply .github/workflows/ci.yml:74]: to make the ci faster

**Review by wfxr — APPROVED** [review]
LGTM

**Pull request merged**